src/components/DeeplinkWrapper/index.website.js

```diff
@@ -23,12 +23,18 @@ const propTypes = {
     ...withLocalizePropTypes,
 };
 
+const desktopAppState = {
+    checking: 'checking',
+    installed: 'installed',
+    notInstalled: 'not-installed',
+};
```

Comment:
Consts should be defined in CONST.js, all uppercase for naming

Reply:
Done